fe/fe-core/src/main/java/org/apache/doris/alter/RollupJobV2.java

```diff
@@ -641,6 +645,12 @@ protected boolean cancelImpl(String errMsg) {
             return false;
         }
 
+        try {
+            this.watermarkTxnId =
+                    Env.getCurrentGlobalTransactionMgr().getNextTransactionId();
+        } catch (Exception e) {
+            LOG.warn("get next transaction id failed");
+        }
```

Comment:
We could put this field into the abstract class AlterJobV2, and add an unified assign method in it, rather than seperate it in each derived class and assign it in duplicated way.

Reply:
done